src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductAttributeValueRepository.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ProductBundle\Doctrine\ORM;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Product\Repository\ProductAttributeValueRepositoryInterface;
+
+class ProductAttributeValueRepository extends EntityRepository implements ProductAttributeValueRepositoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function findByJsonChoiceKey(string $choiceKey): array
+    {
+        return $this->createQueryBuilder('o')
+            ->andWhere('o.json LIKE :key')
+            ->setParameter('key', '%' . $choiceKey . '%')
```

Comment:
I'm wondering if we can add some additional chars to this parameter (like `%"$key"%`)